Week2task6/src/SetAsVector.java

```diff
@@ -0,0 +1,28 @@
+import java.util.Vector;
+import java.util.Collections;
+import java.util.List;
+
+public class SetAsVector extends Vector {
+	public SetAsVector() {
+		super();
+	}
+	public boolean contains(String s) {
+		if (super.contains(s)) {
+			return true;
+		}
+		return false;
+	}
+	public boolean add(String s) {
+		if (super.contains(s)) {
+			return false;
+		}
+		super.add(s);
+			return true;
+	}
+	public boolean remove(String s) {
```

Comment:
It was a misunderstanding. The remove method should return true when it succeeded to remove and false when it did not succeed to remove. So in if super.contains after you do super.remove you shold return true and at the end return false.